### ISSUE TITLE
Полный список особенностей

### ISSUE THREAD

**Issue by aka-sektor** [abramcumner/xray15#3]
Не не увидел в вики, описания изменений.

Предлагается по коммитам смотреть че тут есть? Или все описано в релизах: https://github.com/abramcumner/xray15/releases

**Comment by abramcumner** [comment]
> Предлагается по коммитам смотреть че тут есть? Или все описано в релизах: https://github.com/abramcumner/xray15/releases

Да, сейчас только по коммитам. И когда будет релиз, будет и описание.

**Comment by aka-sektor** [comment]
Ну а навскидку, описать основные фичи до 1.5.10-107 можешь?

**Comment by abramcumner** [comment]
Вближайшее время сделаю релиз с описанием, как раз начало месяца.

**Comment by aka-sektor** [comment]
![1486586365137633626](https://user-images.githubusercontent.com/25452748/42155608-882e801a-7df1-11e8-8e0b-57cb119c4300.png)

**Comment by aka-sektor** [comment]
И всё таки, в релизах кажется важные коммиты.

Вот про версию 1.5.10-08 не сказано, хотя думаю из коммитов между 8 и 20 февралем, можно выделить:
- регулировка радиуса отрисовки травы от КД

А после 1.5.10-12 уже:
- редактор погоды
- воллмарки (блудмарки ?)
- тени от травы
- тень от ГГ в виде от первого лица и консольная команда r__actor_shadow
- настройка худа
- сервера GSC вместо GameSpy
- убрана привязка процесса игры к первому ядру (иными словами "улучшенная многопоточность" ?)
- редактор UI
- обновлен LuaJit до 2.0.5+
- заменен луабинд
- удалены boost и stlport
- добавлена х64 архитектура (иными словами "рабочий 64-битный движок" ?)

**Comment by Xottab-DUTY** [comment]
> улучшенная многопоточность

Не путайте многопоточность и многоядерность. Отвязка от одного ядра позволяет системе перекидывать движок по ядрам процессора в зависимости от их загрузки.
Движок от этого многопоточным не стал и остался однопоточным. (Хотя, там и есть второй поток, но он не особо напрягается)

**Comment by aka-sektor** [comment]
> Не путайте

Сказанное касается ток этого репо? А что насчет OGSR?

**Comment by Xottab-DUTY** [comment]
Что именно? Многопоточность и многоядерность – разные понятия вообще, а не только для этого репозитория ;)

А если ты про остальную часть моего сообщения, то это касается всего X-Ray в целом и его форков (читай – других репозиториев). Пока нет такого репозитория, где движок был бы полностью многопоточным. Да, в каких-то репозиториях ситуация с потоками чуть лучше, чем в других, где-то чуть похуже. Но глобально – движок не изменился и многопоточным не стал.

А многоядерность, да, там достаточно удалить одну строчку и она будет. =)

**Comment by aka-sektor** [comment]
https://www.youtube.com/watch?v=LVx4ZNaLYNg
